package/metadata.json

```
@@ -0,0 +1,25 @@
+{
+    "KPlugin": {
+        "Authors": [
+            {
+                "Email": "mvourlakos@gmail.com",
+                "Name": "Michail Vourlakos"
+            }
+        ],
+        "Category": "Windows and Tasks",
+        "Description": "Window buttons for your panels",
+        "Icon": "focus-windows",
+        "Id": "org.kde.windowbuttons",
+        "License": "GPLv2",
+        "Name": "Window Buttons",
+        "ServiceTypes": [
+            "Plasma/Applet"
+        ],
```

```suggestion
{
    "KPackageStructure": "Plasma/Applet",
    "KPlugin": {
        "Authors": [
            {
                "Email": "mvourlakos@gmail.com",
                "Name": "Michail Vourlakos"
            }
        ],
        "Category": "Windows and Tasks",
        "Description": "Window buttons for your panels",
        "Icon": "focus-windows",
        "Id": "org.kde.windowbuttons",
        "License": "GPLv2",
        "Name": "Window Buttons",
```

https://develop.kde.org/docs/plasma/widget/porting_kf6/#porting-an-existing-plasmoid

> the KPlugin section may still contain the ServiceTypes key. This needs to be replaced by a KPackageStructure entry in the json's top level.

---

thanks